addons/stock/models/stock_rule.py

```diff
@@ -537,6 +538,8 @@ def run_scheduler(self, use_new_cursor=False, company_id=False):
                 self = self.with_env(self.env(cr=cr))  # TDE FIXME
 
             self._run_scheduler_tasks(use_new_cursor=use_new_cursor, company_id=company_id)
+        except Exception:
+            _logger.error('Error during stock scheduler :\n%s' % traceback.format_exc())
```

Comment:
Wait, shouldn't you re-raise the exception?

Reply:
@rco-odoo It is fixed.